.github/workflows/python-app.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.10
+      - name: Set up Python 3.11
         uses: actions/setup-python@v3
         with:
           python-version: "3.11"
```

Comment:
The swap is not needed anymore, in one of my first implementations I had some big memory constraint but can be removed if you want.